app/controllers/sign_and_submit_controller.rb

```diff
@@ -1,14 +1,4 @@
 # frozen_string_literal: true
 
 class SignAndSubmitController < StandardStepsController
```

Comment:
same as above! We are using it, but `update` does not have special logic so we can fall back on `StandardStepsController` -- thanks citizen code, for that!